pkg/storage/SQL/operational_solution/get_by_id.sql

```diff
@@ -16,8 +16,11 @@ SELECT
     OpSol.created_by,
     OpSol.created_dts,
     OpSol.modified_by,
-    OpSol.modified_dts
+    OpSol.modified_dts,
+    PNSL.id IS NOT NULL AS is_common_solution
```

Comment:
Continuing from my previous note, I think comments here would be useful. Looking over this, I am not exactly sure why `is_common_solution` is based no whether or not a Possible Need Solution Link's ID isn't NULL (writing that out shows how complex this can be!)

Reply:
Good call! Thanks Clay, I'll add the comments elsewhere as well.